packages/jest-runner/src/jest-runner.ts

```diff
@@ -182,12 +181,11 @@ class JestRunner implements TestRunner {
             collectCoverage: inExecutionPhase && !!process.env.TAS_COLLECT_COVERAGE
         };
         if (inExecutionPhase) {
-            const { globalConfig, configs } = await readConfigs(jestArgv, projectRoots);
+            const { globalConfig, projectConfig } = await readConfig(jestArgv, Util.REPO_ROOT);
```

Comment:
Not clear with the fix. Because earlier we are getting array of configs and not we are fetching config value only.